CMakeLists.txt

```diff
@@ -38,13 +38,13 @@ SET(SOURCE_FILES
 
 ADD_CUSTOM_COMMAND(
     OUTPUT  "${CMAKE_BINARY_DIR}/quicly-probes.d"
-    COMMAND curl -sL 'https://raw.githubusercontent.com/h2o/h2o/fb0d55a/deps/quicly/quicly-probes.d' >${CMAKE_BINARY_DIR}/quicly-probes.d
+    COMMAND curl -sLf 'https://raw.githubusercontent.com/h2o/h2o/fb0d55a/deps/quicly/quicly-probes.d' >${CMAKE_BINARY_DIR}/quicly-probes.d
 )
 LIST(APPEND DTRACE_FILES "${CMAKE_BINARY_DIR}/quicly-probes.d")
 
 ADD_CUSTOM_COMMAND(
     OUTPUT  "${CMAKE_BINARY_DIR}/h2o-probes.d"
-    COMMAND curl -sLO 'https://raw.githubusercontent.com/h2o/h2o/master/h2o-probes.d' >${CMAKE_BINARY_DIR}/h2o-probes.d
+    COMMAND curl -sLf 'https://raw.githubusercontent.com/h2o/h2o/master/h2o-probes.d' >${CMAKE_BINARY_DIR}/h2o-probes.d
```

Comment:
I'm not sure why `-O` is set here, though 🤔

Reply:
Good idea, given the more than occasional blips :+1: